lambda/translprim.ml

```diff
@@ -522,6 +522,7 @@ let lookup_primitive loc ~poly_mode ~poly_sort pos p =
     | "%floatarray_unsafe_set" ->
       Primitive ((Parraysetu (Pfloatarray_set, Ptagged_int_index)), 3)
     | "%obj_is_int" -> Primitive (Pisint { variant_only = false }, 1)
+    | "%is_null" -> Primitive (Pisnull, 1)
```

Comment:
Do you really need to expose the primitive ? I suppose it makes sense to have it for testing, but I'm assuming that users should always the pattern-matching on the `or_null` type (or other nullable types in the future).

Reply:
Yeah, it's just for testing.